package.json

```diff
@@ -7,7 +7,7 @@
   },
   "scripts": {
     "dev": "concurrently \"yarn start\" \"yarn start-backend\"",
-    "start": "yarn workspace app start",
+    "start": "yarn workspace app start & yarn workspace backend start",
```

Comment:
I don't understand this change. Doesn't it start a dev server? Why don't we use the same command/entrypoint as is in the `Dockerfile`?

Reply:
Yes, this wasn't good solution. Added `.s2i/bin/run` where I am using same entrypoint as is in the `Dockerfile`